client/src/charts/canada/CanadaD3Component.js

```diff
@@ -248,10 +243,7 @@ export class CanadaD3Component {
       });
     const hide_optional_components = (prov_keys, selector_template) =>
       _.each(prov_keys, (prov_key) => {
-        const corresponding_province_has_data = _.some(
-          data,
-          (yearly_data) => yearly_data[prov_key]
-        );
+        const corresponding_province_has_data = data[prov_key];
```

Comment:
We could probably remove this line and adjust the line under `if (!data[prov_key])`